proto/optional_output_wrapper/BUILD

```diff
@@ -0,0 +1,7 @@
+load("//rust:rust.bzl", "rust_binary")

```

Comment:
This is a simple 1 file binary, what is the logic for moving it to a new package?

Reply:
I just find it's find it's cleaner to have this in a separate directory with it's own build file.
Moved it to the same package as the parent.
Ideally we should move it to //proto/private I feel.